starter-code/src/app/component/homepage/homepage.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { PeopleService } from '../../services/people.service'
+
+@Component({
+  selector: 'app-homepage',
+  templateUrl: './homepage.component.html',
+  styleUrls: ['./homepage.component.css']
+})
+export class HomepageComponent implements OnInit {
+  peopleService: PeopleService;
+
+  constructor(peopleService: PeopleService) {
+    this.peopleService = peopleService;
```

Comment:
Instead of this, you can use the shortcut:
```
constructor(private peopleService: PeopleService) {}
```
That said, you are not using that service in this component at all.